docs/changelog.adoc

```diff
@@ -52,6 +52,31 @@ For more information on features and bug fixes in 0.3.0, see the GitHub mileston
 
 * https://github.com/JanusGraph/janusgraph/milestone/4?closed=1
 
+=== Version 0.2.2 (Release Date: October 2, 2018)
+
+[source, xml]
+<dependency>
+   <groupId>org.janusgraph</groupId>
+   <artifactId>janusgraph-core</artifactId>
+   <version>0.2.2</version>
```

Comment:
This should be 0.3.1

Reply:
NVM I see what is happening here

Reply:
yeah, the diff is a bit confusing since I pulled in the 0.2.2 release notes